intranet/static/js/eighth/signupUI.js

```diff
@@ -150,17 +152,19 @@ function stickyHeaders(headers) {
                 thrdHeader = headers.eq(i - 2);
             var top = thisHeader.position().top;
 
-            if(prevHeader.hasClass("empty")) {
+            if (prevHeader.hasClass("empty")) {
                 prevHeader = thrdHeader;
             }
 
             if (top <= 0) {
-                if(!thisHeader.hasClass("empty")) {
+                if (!thisHeader.hasClass("empty")) {
                     $("#" + thisHeader.attr("id") + "-stuck").removeClass("hidden");
                 }
                 headers.each(function(j) {
                     var loopHeader = $(this);
-                    if(j != i) $("#" + loopHeader.attr("id") + "-stuck").addClass("hidden");
+                    if(j !== i) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keyword "if" must be followed by whitespace.](https://www.codacy.com/app/tjcsl/ion/file/2692305614/issues/source?bid=3176160&fileBranchId=3299251#l165)